jenkins/scripts/integration_clean.sh

```diff
@@ -22,10 +22,17 @@ function cleanup() {
     startswith("ci-test-vm-")) | select((.Name | ltrimstr("ci-test-vm-") |
     split("-") | .[0] | strptime("%Y%m%d%H%M%S") | mktime) < (now - 21600))
     | .ID ')
+
+  DEBUGGING_VM_LIST=$(openstack server list -f json | jq -r '.[] | select(.Name |
+  startswith("ci-test-vm-")) | select((.Name | ltrimstr("ci-test-keep-vm-") |
+  split("-") | .[0] | strptime("%Y%m%d%H%M%S") | mktime) < (now - 86400))
+  | .ID ')
   
+  FINAL_LIST=$(IFS=$'\n'; echo "${VM_LIST[*]}" "${DEBUGGING_VM_LIST[*]}")
```

Comment:
I should admit I am bad with naming of vars. If you have better option please let me know